bundles/org.openhab.binding.amberelectric/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amberelectric"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="service">
+		<label>Amber Electric</label>
+		<description>Amber Electric - wholesale access to power prices</description>
+
+		<channels>
+			<channel id="electricity-price" typeId="electricity-price"/>
+			<channel id="electricity-status" typeId="electricity-status"/>
+			<channel id="controlled-load-price" typeId="controlled-load-price"/>
+			<channel id="controlled-load-status" typeId="controlled-load-status"/>
+			<channel id="feed-in-price" typeId="feed-in-price"/>
+			<channel id="feed-in-status" typeId="feed-in-status"/>
+			<channel id="nem-time" typeId="nemtime"/>
+			<channel id="spike" typeId="spike"/>
+			<channel id="renewables" typeId="renewables"/>
+		</channels>
+
+		<config-description>
+			<parameter name="apiKey" type="text" required="true">
+				<label>API Key</label>
+				<description>API key from the Amber website</description>
+			</parameter>
+			<parameter name="nmi" type="text">
+				<label>NMI</label>
+				<description>NMI for your address (Optional)</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="60" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="electricity-price">
+		<item-type>Number:EnergyPrice</item-type>
+		<label>Current Electricity Price</label>
+		<description>Current price to import power from the grid</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="controlled-load-price">
+		<item-type>Number:EnergyPrice</item-type>
+		<label>Current Controlled Load Price</label>
+		<description>Current price to import power for Controlled Load</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="feed-in-price">
+		<item-type>Number:EnergyPrice</item-type>
+		<label>Current Feed-In Price</label>
+		<description>Current price to export power to the grid</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
```

Comment:
It seems these channel types are very similar to `electricity-price`. So you could drop them and just override the labels and descriptions for the corresponding channel definitions. Thereby you would reduce redundancy.

Reply:
This would be cool, do you know any bindings I can review to see how this is done?I grep'd on override and didn't find anything

Reply:
Sure! There are many to choose from, but here's a related one:
https://github.com/openhab/openhab-addons/blob/f748c18cde9688b5670ef04c2ca8d5d0f76ee17c/bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/thing/channel-types.xml#L15-L22

and the channels referencing this channel type and overriding label/description:
https://github.com/openhab/openhab-addons/blob/f748c18cde9688b5670ef04c2ca8d5d0f76ee17c/bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/thing/channel-groups.xml#L15-L34

Reply:
Brilliant commit incoming.